workflow-templates/im-build-npm-package.yml

```diff
@@ -94,14 +96,19 @@ jobs:
       #     read-pkg-token: ${{ secrets.READ_PKG_TOKEN }} # This is an org-level secret
       #     orgs: 'im-client,im-enrollment,im-practices' # TODO:  Verify list of orgs packages will be pulled from
 
+      # TODO:  If your project contains a .npmrc file that points to any of our im-* orgs as a registry you have two options:
+      #        1 - Remove the .npmrc file completely and ensure you are using the im-open/authenticate-with-gh-package-registries action in each job that needs to authenticate
+      #        2 - Supply a READ_PKG_TOKEN environment variable to each step that might use the .npmrc file for authentication with GH Packages (dotnet build, setup node, npm ci, etc)
```

Comment:
It feels a little off to me to have this note below a step that is using the `authenticate-with-gh-package-registries` action. Maybe if we referenced the fact that it's above like "... ensure you are using the im-open/authenticate-with-gh-package-registries action in each job that needs to authenticate, like above", or combine this note with the block above?